src/main/java/Main/StreamingService.java

```diff
@@ -37,9 +37,10 @@ public Set<Media> searchMedia(){
             for (Media m : media) {
                 if(m.getTitle().contains(mediaName)) {
                     result.add(m);
-                    TextUI.displayMsg(mediaName + " found, enjoy!");
                 }
             }
+            TextUI.displayMsg(mediaName + " found, enjoy!");
```

Comment:
Kan denne linje printe alle resultater ud? Hvis jeg søger på "The", vil jeg jo forvente at den printer alle film/serier ud der indeholder "The" i titlen.